format/puffin-spec.md

```diff
@@ -123,6 +123,44 @@ The blob metadata for this blob may include following properties:
 
 - `ndv`: estimate of number of distinct values, derived from the sketch.
 
+#### `delete-vector-v1` blob type
+
+A serialized delete vector that represents the positions of rows in a file that
+are deleted.  A set bit at position P indicates that the row at position P is
+deleted.
+
+The bitmap supports positive 64-bit positions, but is optimized for cases where
```

Comment:
The `vector`? Not sure if we want to use bitmap here.

The vector supports positive 64-bit positions, but the physical layout is optimized for cases 
in which most positions fit within 32 bits. Positions are divided into 32-bit keys (most significant 4 bytes - do we need to mention endiness here?) and a 32-bit unsigned integer sub-position (some other name here?).  Each key maps to a 32-bit Roaring bitmap which will have the sub-position bit set.


The thing I mostly want to avoid here is using "position" in 2 contexts, once for in the file and once for in the 32-bit roaring bitmap.

Reply:
I agree with using `vector` here instead of bitmap since bitmap is relevant for the serialized representation which is explained further below, or @RussellSpitzer explanation structure also sounds good to me. 

>The thing I mostly want to avoid here is using "position" in 2 contexts, once for in the file and once for in the 32-bit roaring bitmap.

For this, I feel like the explanation that's further below is OK since it draws a distinction between a "key" and the 4 least significant bytes. And then later on for the offset of the blob in the puffin file, I think we'd use the word "offset".

Reply:
Oh then I think we have 3 instances, I meant

64 bit Position (Global position in data file)
32 bit position (Part of the position in teh data file, but not the whole thing)
and Position within Puffin File

Reply:
Vector vs bitmap: Updated to use "vector" but clarify above that "bitmap" is often used to mean the same thing.
Position: Updated to use "sub-position" for the 32-bit portion. The position in Puffin is referred to as the "offset".
Endianness: No need to mention because "most significant bytes" is clear and independent of order.